docs/source/contributors/devinstall.md

```diff
@@ -28,53 +28,28 @@ Enterprise Gateway's build environment is centered around `make` and the corresp
 Entering `make` with no parameters yields the following:
 
 ```
-activate                       Print instructions to activate the virtualenv (default: enterprise-gateway-dev)
 clean-images                   Remove docker images (includes kernel-based images)
 clean-kernel-images            Remove kernel-based images
 clean                          Make a clean source tree
-dev                            Make a server in jupyter_websocket mode
 dist                           Make source, binary, kernelspecs and helm chart distributions to dist folder
 docker-images                  Build docker images (includes kernel-based images)
 docs                           Make HTML documentation
-env                            Make a dev environment
 helm-chart                     Make helm chart distribution
-install                        Make a conda env with dist/jupyter_enterprise_gateway-*.whl and dist/jupyter_enterprise_gateway-*.tar.gz installed
```

Comment:
Are we removing `install`? Or just hiding it?

Reply:
Per the PR description, it has been renamed to `test-install`, since it does nothing more than that.